app/_src/guides/builtin-gateway.md

```diff
@@ -0,0 +1,340 @@
+  name: my-gateway
+spec:
+  selectors:
+    - match:
+        kuma.io/service: edge-gateway
+  conf:
+    listeners:
+      - port: 8080
+        protocol: HTTP
+        tags:
+          port: http/8080
+" | kubectl apply -f -
+```
+
+Notice how the selector selects the `kuma.io/service` tag of the previously defined `MeshGatewayInstance`.
+
+Now that you have this running you can see gateway pods running:
+
+```shell
+$ kubectl get pods -n kuma-demo
+NAME                            READY   STATUS    RESTARTS   AGE
+redis-5fdb98848c-5tw62          2/2     Running   0          5m5s
+demo-app-c7cd6588b-rtwlj        2/2     Running   0          5m5s
+edge-gateway-66c76fd477-ncsp5   1/1     Running   0          18s
+edge-gateway-66c76fd477-thxqj   1/1     Running   0          18s
+```
+
+You can then retrieve the public url for your gateway with:
+
+```shell
+export PROXY_IP=$(kubectl get svc --namespace kuma-demo edge-gateway -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
+echo $PROXY_IP
+```
+
+You can check the gateway is running:
+
+```shell
+$ curl -v ${PROXY_IP}:8080
+*   Trying 127.0.0.1:8080...
+* Connected to 127.0.0.1 (127.0.0.1) port 8080
+> GET / HTTP/1.1
+> Host: 127.0.0.1:8080
+> User-Agent: curl/8.4.0
+> Accept: */*
+>
+< HTTP/1.1 404 Not Found
+< content-length: 62
+< content-type: text/plain
+< vary: Accept-Encoding
+< date: Fri, 09 Feb 2024 10:07:26 GMT
+< server: Kuma Gateway
+<
+This is a Kuma MeshGateway. No routes match this MeshGateway!
+```
+
+We can see a default response that says there are not routes configured.
+
+## Define a route using `MeshHTTPRoute`
+
+[`MeshHTTPRoute`](/docs/{{ page.version }}/policies/meshhttproute/) lets you define http routes inside your service mesh.
+You can attach a route to an entire gateway or to a single listener by using the `targetRef.kind: MeshGateway` 
+
+```shell
+echo "
+apiVersion: kuma.io/v1alpha1
+kind: MeshHTTPRoute
+metadata:
+ name: edge-gateway-route
+ namespace: {{site.mesh_namespace}} 
+ labels:
+   kuma.io/mesh: default
+spec:
+ targetRef:
+   kind: MeshGateway
+   name: my-gateway
+ to:
+ - targetRef:
+     kind: Mesh
+   rules:
+   - matches:
+     - path:
+         type: PathPrefix
+         value: "/"
+     default:
+       backendRefs:
+       - kind: MeshService
+         name: demo-app_kuma-demo_svc_5000
+" | kubectl apply -f -
+```
+
+Now if we try to reach our gateway again: 
+```shell
+$ curl -v ${PROXY_IP}:8080
+*   Trying 127.0.0.1:8080...
+* Connected to 127.0.0.1 (127.0.0.1) port 8080
+> GET / HTTP/1.1
+> Host: 127.0.0.1:8080
+> User-Agent: curl/8.4.0
+> Accept: */*
+>
+< HTTP/1.1 403 Forbidden
+< content-length: 19
+< content-type: text/plain
+< date: Fri, 09 Feb 2024 10:10:16 GMT
+< server: Kuma Gateway
+< x-envoy-upstream-service-time: 24
+<
+* Connection #0 to host 127.0.0.1 left intact
+RBAC: access denied%
+```
+
+We see we're getting a forbidden error. This is because we do have permissions from the gateway to talk to our demo-app service.
```

Comment:
**Suggestion**: language

```suggestion
We see we're getting a forbidden error. This is because we don't have permissions from the gateway to talk to our demo-app service.
```